README.md

```diff
@@ -355,7 +355,6 @@ Have something to contribute or discuss? [Open a pull request](https://github.co
         <ul>
           <li><a href="https://docs.bazel.build/versions/master/be/java.html">https://docs.bazel.build/versions/master/be/java.html</a></li>
         </ul>
-        </ul>
```

Comment:
Not relevant to the change itself but I’ve noticed it sticking out on the website.

Reply:
Thanks.